# Defines a 2D classification result.
#
float64 score
int16 class
